alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -149,12 +183,10 @@ class SecureMonitorRestApiIT : AlertingRestTestCase() {
                 )
             )
             userClient?.makeRequest("POST", ALERTING_BASE_URI, emptyMap(), monitor.toHttpEntity())
-            fail("Expected 403 Method FORBIDDEN response")
```

Comment:
Comment out this test since it is affected by the bug where its hitting the `fail` and fix it later

Reply:
done